packages/target-nodejs-sdk/src/target.js

```diff
@@ -122,7 +122,13 @@ export function executeDelivery(options, decisioningEngine) {
   );
 
   return deliveryMethod
-    .execute(organizationId, sessionId, deliveryRequest, config.version)
+    .execute(
+      organizationId,
+      sessionId,
+      deliveryRequest,
+      config.version,
+      telemetryProvider
+    )
     .then((response = {}) => {
       logger.debug(
```

Comment:
I think this would be an appropriate spot to call telemetryProvider.addEntry().  We have access to the response as well as the deliveryRequest and config.  So we should have all the data we need to create a telemetry entry.
For calculating the execution time we can use perftool.timeStart/timeEnd just like in decisionProvider.js